packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -98,6 +99,44 @@ export interface IMockContainerRuntimePendingMessage {
 	localOpMetadata: unknown;
 }
 
+/**
+ * Options for the container runtime mock.
+ */
+export interface MockContainerRuntimeOptions {
+	/**
+	 * Sets the flush mode for the runtime. In Immediate flush mode the runtime will immediately
```

Comment:
First sentence makes me think this might be mutable, which is untrue. Maybe just "Flush mode for the runtime." You could also mark the option `readonly`, though readonly-ness in TS is pretty dubious IMO.